Exception calling ".ctor" with "1" argument(s):

```
Run /home/runner/work/_actions/oliver-butterley/lean-update/v1-alpha/scripts/getLatest.ps1
ForEach-Object: /home/runner/work/_actions/oliver-butterley/lean-update/v1-alpha/scripts/getLatest.ps1:10
Line |
  10 |  … $versions | ForEach-Object { [System.Management.Automation.SemanticVe …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling ".ctor" with "1" argument(s): "The input string '' was
     | not in a correct format."
Error: Process completed with exit code 1.
```
see: https://github.com/Seasawher/mathlib4-tactics/actions/runs/11176249532/job/31069206463

--- a/scripts/getLatest.ps1
+++ b/scripts/getLatest.ps1
@@ -1,17 +1,20 @@
 # get all release tags from leanprover/lean4
+# and filter out the ones that are not form of `v*`
 $versions = gh release list `
   --repo leanprover/lean4 `
   --json tagName | `
   ConvertFrom-Json | `
   ForEach-Object { $_.tagName } | `
-  ForEach-Object { $_ -replace '^v' }
+  Where-Object { $_ -like 'v*' }
 
 # parse the version tags as semver
-$semvers = $versions | ForEach-Object { [System.Management.Automation.SemanticVersion]::new($_) }
+$semvers = $versions | `
+  ForEach-Object { $_ -replace '^v' } | `
+  ForEach-Object { [System.Management.Automation.SemanticVersion]::new($_) }
 
 # sort the versions and get the latest one
 $latest = $semvers | Sort-Object | Select-Object -Last 1
-Write-Host "Latest Lean release is: $latest"
+Write-Host "Latest Lean release is: v$latest"
 
 # update `lean-toolchain` file
 $leanStyleVersion = "leanprover/lean4:v$latest"

